pyproject.toml

```
@@ -52,7 +51,6 @@ doc = [
     "numpydoc",
     "pandas",
     "gitpython",
```

```suggestion
    "gitpython",
    "pylint",
```

otherwise the docs build will be broken: https://app.readthedocs.org/projects/power-grid-model/builds/28366968/

see also github.com/PowerGridModel/power-grid-model/pull/1000#discussion_r2116226974